torch/_inductor/codegen/triton.py

```diff
@@ -750,10 +750,10 @@ def _split_iteration_ranges(
 
         def add_range(i, expr):
             expr = sv.simplify(expr)
-            if not sv.maybe_guard_multiple_of(remaining[i], expr):
+            if not sv.statically_known_multiple_of(remaining[i], expr):
                 raise CantSplit()
             # guard on the last item out
-            sv.maybe_guard_equals(remaining[i], expr)
+            sv.statically_known_equals(remaining[i], expr)
```

Comment:
This seems pretty suspect, the comment implies a guard must be installed, and I'm pretty sure the new formulation is literally a noop now.

Reply:
Correct - this has not been audited, see below.